MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -207,9 +215,11 @@ private static IEnumerable<Directory> ProcessChunk([NotNull] PngChunk chunk)
             else if (chunkType == PngChunkType.tEXt)
             {
                 var reader = new SequentialByteArrayReader(bytes);
-                var keyword = reader.GetNullTerminatedString(maxLengthBytes: 79);
+                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(defaultEncoding);
                 var bytesLeft = bytes.Length - keyword.Length - 1;
```

Comment:
Yes, probably (without testing it). In the previous code, it only ever returned a string so there was nothing else to go on. StringValue has a Length that counts the bytes, which is better anyway:

`var keywordStringValue = reader.GetNullTerminatedStringValue(maxLengthBytes: 79);`
`var bytesLeft = bytes.Length - keywordStringValue.Length - 1;`

Subsequent places after that use `keyword` could use `keywordStringValue.ToString([some encoding])` instead

(untested on my end)